Kots Release 1.56.0 (#604)
replicatedhq/kots.io: content/release-notes/1.56.0.md

---
date: 2021-11-04T10:00:00-05:00
linktitle: "1.56.0"
title: "1.56.0"
kubernetes: "1.19, 1.20, and 1.21"
---

{{<features>}}

> * Added the ability to view some detailed information (such as logs, events, etc.) for each unhealthy pod, in the "Support bundle analysis" view under the "Troubleshoot" tab in the admin console.

Reviewer: ```suggestion
* From the **Troubleshoot** tab in the admin console, you can view some detailed information (such as logs, events, etc.) for each unhealthy pod in the **Support bundle analysis** view.
```
